packages/dd-trace/src/appsec/reporter.js

```diff
@@ -18,24 +18,33 @@ let limiter = new Limiter(100)
 const metricsQueue = new Map()
 
 const contentHeaderList = [
-  'content-encoding',
-  'content-language',
   'content-length',
-  'content-type'
+  'content-type',
```

Comment:
You're changing here alphabetical order by another. I think that there is a reason like sync with some documentation or something.

Add a comment here please, to prevent next developer changing the order to alphabetical order again

Reply:
fixed